metadata.rb

```diff
@@ -1,8 +1,15 @@
-name             'timesync'
-maintainer       'Oregon State University'
+name 'timesync'
```

Comment:
We don't normally run rubocop on `metadata.rb`. See [our usual config](https://github.com/osuosl-cookbooks/osl-mirror/blob/master/.rubocop.yml)